src/Recommendations/Rule/MostRecent.php

```diff
@@ -65,14 +66,8 @@ public function addRelations(RuleModel $model, array $list): array
     public function supports(): array
     {
         return [
-            'correction',
```

Comment:
this is an improvement for clarity, but the kind of articles not supported here are already excluded by the query in `RuleModelRepository`